app/src/hooks.client.ts

```diff
@@ -1,25 +1,25 @@
-import { handleErrorWithSentry } from '@sentry/sveltekit';
+import { captureException } from '@sentry/sveltekit';
 import { error as logErrorToFile } from 'tauri-plugin-log-api';
-import type { NavigationEvent } from '@sveltejs/kit';
+import type { HandleClientError } from '@sveltejs/kit';
 
-function myErrorHandler({ error, event }: { error: any; event: NavigationEvent }) {
-	console.error(error.message + '\n' + error.stack);
-	logErrorToFile(error.toString());
-	console.error('An error occurred on the client side:', error, event);
-}
+// eslint-disable-next-line func-style
+export const handleError: HandleClientError = ({ error, status }) => {
+	let errorId: string = crypto.randomUUID();
```

Comment:
On Safari this crypto method [seems to only be available from March 2022](https://caniuse.com/mdn-api_crypto_randomuuid), is there a more compatible way we could generate this ID?

Reply:
So it was added in Safari `15.2` it looks like, right. It seems 99%+ of our users are on Safari 16+, check out: https://eu.posthog.com/project/2812/insights/PpsNLmf7

That top 52.9% are seemingly using some useragent cloaking privacy feature of Mac OS X / Safari, but Caleb on his up-to-date Macbook had the same useragent. The rest are Windows users with Edge webview / Chromium 126 (latest version), and then Linux folks with webkitgtk that seems to advertise as Safari 17 / Safari 16.

So long story short I think we're probably good here. But, if you want to be on the safe side, this ID doesn't really need to be super collision resistant, right, plus it'll most likely be overwritten by the Sentry generated ID anyway, so I could swap it out with some `Math.random()`-based solution, just let me know :+1:

Reply:
This might be related to the top 52.9%: https://chromestatus.com/feature/5452592194781184
> Cap the reported macOS version in the user-agent string at 10_15_7

I also had a look at the data and it seems to me that we don't actually have any version information available for macos? The strings _with_ version data all appear to be from linux platforms.

@PavelLaptev do you recall what the oldest version is we have encountered wrt to CSS bugs?

Reply:
Yeah interesting, so all Mac platforms seem to be wrapped up in that one capped user-agent string :thinking: 

I did a bit more research, and it seems that there's often another suffix on that user-agent string we're dealing with which includes more info (See for example: https://github.com/nielsbasjes/yauaa/blob/main/analyzer/src/main/resources/UserAgents/Apple.yaml#L1704). But that seems totally missing in our case.

Anyway, gonna to merge this PR now :+1: